src/NaluParsing.C

```diff
@@ -167,8 +167,6 @@ namespace sierra
         }
       }
 
-      const YAML::Node& oversetUserData = node["overset_user_data"];
```

Comment:
This breaks builds with TIOGA enabled

Reply:
Ah oh yeah.  Okay I'll move this inside the `#ifdef`

Reply:
K pushed a fix for this.  I've added the WIP tag to prevent merging until I finish testing the cuda build.